File: ECommerceAPI/Models/Storage.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ECommerceAPI.Models
{
    public partial class Storage
    {
        public int StorageId { get; set; }
        public string StorageName { get; set; }
        public int? SupplierId { get; set; }
        public int? ProductId { get; set; }
        public int? Quantity { get; set; }

        public virtual Product Product { get; set; }
        public virtual Supplier Supplier { get; set; }
    }
}

File: ECommerce-Bill_Feature/src/app/models/Storage.ts
import { Product } from "./Product";
import { Supplier } from "./Supplier";

export class Storage {
    storageId: number;
    storageName: string;
    supplierId: number | null;
    productId: number | null;
    quantity: number | null;
    product: Product;
    supplier: Supplier;
}